python/src/aiconfig/registry.py

```diff
@@ -74,7 +74,7 @@ def get_model_parser_for_prompt(prompt: Prompt, config: "AIConfigRuntime"):
         Returns:
             ModelParser: The retrieved model parser
         """
-        model_name = config.get_model_name(prompt)
+        model_name = config.get_model_name(prompt).strip()
```

Comment:
I think we should just leave this as-is. If it's in the config already then we should have either sanitized it or it was before we started sanitizing it and stripping is going to make it inaccessible.

> Because we could have existing configs that have model names already, I only changed the setter methods, not the getter ones like get_model(), so this is not a breaking change

Reply:
If use does something like this:
```
model_name = "  my_model  "
config.update_model(model_name, None)
# does some stuff...

config.get_model_parser_for_prompt(model_name)
```

This will fail if we don't do the strip, so I think it's important to keep it. I'll re-word the summary

Reply:
Made an issue to surface unfound model/model parser names in https://github.com/lastmile-ai/aiconfig/issues/1136 similar to what we have for prompt error message

Reply:
Can you explain why this will fail? With the other changes in this PR, config.get_model_name(prompt) should not have any extra whitespace already, so no need to strip it, right?